test/Agent.js

```diff
@@ -8,62 +8,61 @@ var expect = chai.expect
 var should = chai.should()
 
 describe('Agent', () => {
-    it('Should identify as client when cast as client', function(){
+    it('Identifies as a client when cast as client', function () {
         var agent = new Agent(UserLib.Client)
         expect(agent.user.constructor.name).to.equal("Client")
         expect(agent.user.type).to.equal(UserType.Client)
     })
 
-    it('Should identify as identity when cast as identity', function(){
+    it('Identifies as an identity when cast as identity', function () {
         var agent = new Agent(UserLib.Identity)
         expect(agent.user.constructor.name).to.equal("Identity")
         expect(agent.user.type).to.equal(UserType.Identity)
     })
 
-    it('Should identify of type Unloq as identity when cast as identity of Unloq', function(){
+    it('Identifies as a Unloq identity when cast as a Unloq identity', function () {
         var agent = new Agent(UserLib.Identity, Unloq, process.env.UNLOQ_KEY)
         expect(agent.user.constructor.name).to.equal("Identity")
         expect(agent.user.type).to.equal(UserType.Identity)
         expect(agent.user.identity.type).to.equal('Unloq')
-        
     })
 
-    it('Should identify as server when cast as server', function(){
+    it('Identifies as a server when cast as server', function () {
         var agent = new Agent(UserLib.Server)
         expect(agent.user.constructor.name).to.equal("Server")
         expect(agent.user.type).to.equal(UserType.Server)
     })
-  
-    it('Should identify as generic when not cast', function(){
+
+    it('Identifies as generic when not cast', function () {
         var agent = new Agent()
         expect(agent.user.constructor.name).to.equal("User")
         expect(agent.user.type).to.equal(UserType.Generic)
     })
 
-    it('Should allow server to generate random', function(){
+    it('Allows a server to generate random', function () {
```

Comment:
While I know this was not in the original test, still ... "allow a server to generate random" .... what? Or is it generate x based on random y? Anyway, a bit more descriptive here will be good.

Reply:
done, Talked with @genecyber, and it looks like it is generating an Envelope object. Updated the test title as such.